frontends/web/src/components/settingsButton/settingsButton.module.css

```diff
@@ -62,6 +70,10 @@
     cursor: default;
 }
 
+.noClick {
+    cursor: default;
+}
```

Comment:
is this needed if the button is just disabled, see me other comment about disabled.

Reply:
I think this is still needed, see my previous comment